### PR TITLE
Add some conventions

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: docs/developer-guide/index.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ All Next applications use `make` and all implement the following `make` commands
 - `make deploy` deploys the application (you may need to environment variables for this to run successfully)
 - `make run` runs the application locally (usually on port `localhost:5050`).
 
+## Conventions
+- We write code in U.S. English.  This is to avoid things like `color: $next-colour-blue;`.
+- We use tabs for indentation.
```
Comment (ironsidevsquincy) [inline docs/developer-guide/index.md]: always preferred spaces, e.g. using tabs makes github view really wide

File: docs/developer-guide/index.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ All Next applications use `make` and all implement the following `make` commands
...
+- We take turns to bring biscuits into office.
```
Comment (kaelig) [inline docs/developer-guide/index.md]: Gluten free plz thx.

File: docs/developer-guide/index.md (hunk 1 above)
Comment (matthew-andrews) [inline docs/developer-guide/index.md]: https://github.com/sindresorhus/github-tab-size

File: docs/developer-guide/index.md (hunk 2 above)
Comment (matthew-andrews) [inline docs/developer-guide/index.md]: We aim to eliminate allergies during Q4.

Comment (commuterjoy) [inline docs/developer-guide/index.md]: https://github.com/Financial-Times/next-bug-rota/issues/5

**Pull request merged**